precompiles/outposts/stride/tx.go

```diff
@@ -131,7 +137,7 @@ func (p Precompile) LiquidStake(
 		return nil, err
 	}
 
-	return method.Outputs.Pack(res.Sequence, true)
+	return method.Outputs.Pack(true)
```

Comment:
Why are we removing the Sequence?

Reply:
I don't see an actual use case for it considering it's targeted for use with the dApp store and not Solidity devs.